### ISSUE TITLE
How to add multiple tags to a droplet during creation?

### ISSUE THREAD

**Issue by dagostinelli** [digitalocean/doctl#183]
I'm trying to assign two tags to a droplet during creation time.  The create command appears to only accept a single tag at a time.  Can you advise how to do this?

`doctl compute droplet create db01 --region nyc3 --size 512mb --image centos-7-0-x64 --enable-private-networking --ssh-keys <something> --wait --tag-name production,database --format ID --no-header`

This does not work.  

What I can do is use a totally different API to do it:

```
#!/bin/sh

echo "create droplet"
ID="$(doctl compute droplet create database --region nyc3 --size 512mb --image centos-7-0-x64 --enable-private-networking --ssh-keys <something> --wait --format ID --no-header)"
echo "droplet is ${ID}"

echo "Tagging Droplet"
doapi-droplet tag loadbalancer "${ID}"
doapi-droplet tag staging "${ID}"
```

I would like to use `doctl` for everything.  What do I do?

**Comment by xmudrii** [comment]
You should use `--tag-names` flag instead of `--tag-name`. First is taking array (multiple tags) while second is taking only one tag.‎Issue #166 should explain to you difference between this two flags and show you example how to use them.

**Comment by dagostinelli** [comment]
Thank you.  It appears that my version of doctl does not have this switch.  

`unknown flag: --tag-names`

I'm on latest.  I guess I need to build from source.

```
$ doctl version
doctl version 1.5.0-release
Git commit hash: 423c7ac
```

**Comment by xmudrii** [comment]
Yes, this is the reason why `--flag-names` flag is unavailable. It has been implemented after 1.5.0 got released. You can wait for 1.6.0 to get released or to build `doctl` from source.

**Comment by dagostinelli** [comment]
That works, I got source and built.  Thank you.
